gcn_monitor/kafka.py

```diff
@@ -38,6 +104,12 @@ def run():
     while True:
         for message in consumer.consume(timeout=1):
             topic = message.topic()
+            key, body = parse_into_s3_object(message)
+            s3_client.put_object(
+                Bucket=bucketarn.replace("arn:aws:s3:::", ""),
```

Comment:
Does boto3 have a method to parse ARNs?

Reply:
None that i've seen from a quick google

Reply:
Since what you really need is the bucket name, not its ARN, why not take the bucket name as the CLI argument?